src/base/conversion.rs

```diff
@@ -118,12 +118,13 @@ macro_rules! impl_from_into_asref_1D(
               S: ContiguousStorage<N, $NRows, $NCols> {
             #[inline]
             fn into(self) -> [N; $SZ] {
-                unsafe {
-                    let mut res: [N; $SZ] = mem::uninitialized();
-                    ptr::copy_nonoverlapping(self.data.ptr(), &mut res[0], $SZ);
+                let mut res = mem::MaybeUninit::<[N; $SZ]>::uninit();
 
-                    res
-                }
+                unsafe { ptr::copy_nonoverlapping(self.data.ptr(), res.as_mut_ptr() as *mut N, $SZ); }
+
+                let res: [N; $SZ] = unsafe { res.assume_init() };
+                
+                res
```

Comment:
Nit: this can just be `unsafe { res.assume_init() }`. No need for the assignment to `res`.